macos/FluentUI/Link.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+/// A lightweight hyperlink control
+@objc(MSFLink)
+open class Link : NSButton {
```

Comment:
Are we just making all controls "open" because someone will eventually want to be able to use an override for standard NSButton override points? Probably ok but just curious what our guiding principle is on that

Reply:
Our default has been "there's no reason to prevent users from subclassing these controls" and there are certainly valid reasons for subclassing most AppKit control types.